Source/Atomic/UI/UIWidget.h

```diff
@@ -250,10 +258,18 @@ class UIWidget : public Object, public tb::TBWidgetDelegate
     bool GetCaptured();
 
     void SetCapturing(bool capturing);
-    bool GetCapturing() { return widget_->GetCapturing(); }
+    bool GetCapturing();
 
     void InvokeShortcut(const String& shortcut);
 
+    bool GetShorten();
```

Comment:
GetShorten/SetShorten are also a bit unclear, could use a doc string, and GetShorten can be const (just noting, I miss plenty of these too)